.github/workflows/ci.yml

```
@@ -1,6 +1,7 @@
-name: sml / main
+name: sml / ci
 
 on:
+  pull_request:
   push:
     branches: [master, main]
```

If you'd like, you could do a follow-up PR to change this to:

```suggestion
    branches: [main]
```

We've moved to using `main` for all repos.